### PR TITLE
CU-1wty1h0 liquidations test/bench/runtime - required by lending

### PR DESCRIPTION
**Pull request by dzmitry-lahoda**

### CODE REVIEW

**Comment by ComposableFiB0t** [comment]
Task linked: [CU-1wty1h0 liqudation: write runtime tests](https://app.clickup.com/t/1wty1h0)

File: runtime/dali/Cargo.toml [hunk 1]
```diff
@@ -174,12 +175,14 @@ std = [
...
 	"bonded-finance/std",
 	"dutch-auction/std",
+	"liquidations/std",
 	"vesting/std",
 	"mosaic/std",
 	"preimage/std",
 	"membership/std",
 	"system-rpc-runtime-api/std",
 	"transaction-payment-rpc-runtime-api/std",
+	"liquidations/std",
```
Comment (vivekvpandya) [inline runtime/dali/Cargo.toml:185]: seems redundant

File: frame/liquidations/src/mock/runtime.rs [hunk 2]
```diff
@@ -0,0 +1,266 @@
...
+}
+
+parameter_types! {
+	pub const SS58Prefix: u8 = 42;
+	pub const BlockHashCount: u64 = 250;
+}
+
+impl frame_system::Config for Runtime {
+	type BaseCallFilter = Everything;
+
```
Comment (vivekvpandya) [inline frame/liquidations/src/mock/runtime.rs:62]: extra white spaces.

File: runtime/dali/Cargo.toml (hunk 1 above)
Comment (dzmitry-lahoda) [reply runtime/dali/Cargo.toml:185]: why? all other pallets set their std here?

Comment (vivekvpandya) [reply runtime/dali/Cargo.toml:185]: yes but line number 178 and 185 are identical , I don't understand why we need both of them?

Comment (dzmitry-lahoda) [reply runtime/dali/Cargo.toml:185]: ahaa:)  thanks
image

Comment (dzmitry-lahoda) [reply runtime/dali/Cargo.toml:185]: just was not visible in uI

**Review by vivekvpandya — APPROVED** [review]

File: frame/liquidations/src/mock/runtime.rs (hunk 2 above)
Comment (dzmitry-lahoda) [reply frame/liquidations/src/mock/runtime.rs:62]: fixed in next pr to unclude lending

**Pull request merged**